runtime/autoload/health/provider.vim

```diff
@@ -522,8 +522,8 @@ function! s:check_node() abort
   let host = provider#node#Detect()
   if empty(host)
     call health#report_warn('Missing "neovim" npm package.',
-          \ ['Run in shell: npm install -g neovim',
-          \  'Is the npm bin directory in $PATH?'])
+          \ ['Run in shell: npm install -g neovim or yarn global add neovim',
```

Comment:
use a separate advice line:

    \ ['Run in shell (if you use yarn): yarn global add neovim',